src_py/cursors.py

```diff
@@ -365,22 +365,22 @@ def bitswap(num):
 
     info = width, height, hotx, hoty
 
-    for line in range(len(curs)):
-        if curs[line].startswith('static char') or curs[line].startswith('static unsigned char'):
+    possible_starts = ['static char', 'static unsigned char']
+    for line in curs:
+        if line.startswith(possible_starts):
             break
 
-    data = ' '.join(curs[line+1:]).replace('};', '').replace(',', ' ')
+    data = ' '.join(curs[i+1:]).replace('};', '').replace(',', ' ')
     cursdata = []
     for x in data.split():
         cursdata.append(bitswap(int(x, 16)))
 
     cursdata = tuple(cursdata)
-
-    for line in range(len(mask)):
-        if mask[line].startswith('static char') or mask[line].startswith('static unsigned char'):
+    for line in mask:
+        if line.startswith(possible_starts):
             break
 
-    data = ' '.join(mask[line+1:]).replace('};', '').replace(',', ' ')
+    data = ' '.join(mask[i+1:]).replace('};', '').replace(',', ' ')
```

Comment:
I don't think this is correct to use i instead of line?

Reply:
Good catch, there's a problem here, there is no enumerate(), I'm going to fix it.

Reply:
I changed `for line in mask:` to ` for i, line in enumerate(mask):` so the `i` now exists.

Reply:
cool :)